jquery-toggle.js

```diff
@@ -103,6 +113,7 @@
 		var active = prev.next();
 		if (!active.length) active = input.children(":first");
 		active.prop('selected', true);
+		active.each((i, elem) => simulateChange(elem));
```

Comment:
This lib could try to use older JS standard. Maybe the file would be compatible with the IE 11 then.